<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1632</width>
    <height>918</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <item row="3" column="0">
           <widget class="QCheckBox" name="Expert3">
            <property name="text">
             <string>Эксперт 3</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QComboBox" name="comboBox">
              <item>
               <property name="text">
                <string>1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>4</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>6</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>7</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>8</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>9</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>10</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>11</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>12</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>13</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>14</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>15</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>16</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>17</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>18</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>19</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>20</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>21</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>22</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>23</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>24</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>25</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>26</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>27</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>28</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>29</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>30</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>31</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>32</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>33</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>34</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>35</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>36</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>37</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>38</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>39</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>40</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="1">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="Expert5">
            <property name="text">
             <string>Эксперт 5</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="6" column="0">
           <widget class="QPushButton" name="loadTest">
            <property name="text">
             <string>Загрузить
 тест</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="Expert4">
            <property name="text">
             <string>Эксперт 4</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="openImage">
            <property name="focusPolicy">
             <enum>Qt::WheelFocus</enum>
            </property>
            <property name="text">
             <string>Открыть
 изображение</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="Expert1">
            <property name="text">
             <string>Эксперт 1</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="Expert2">
            <property name="text">
             <string>Эксперт 2</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Распознанные
    символы</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="buttonSave">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Сохранить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>48</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="sizeConstraint">
             <enum>QLayout::SetNoConstraint</enum>
            </property>
            <item>
             <widget class="QPushButton" name="button1">
              <property name="text">
               <string>а</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button2">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>б</string>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
              <property name="default">
               <bool>false</bool>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button3">
              <property name="text">
               <string>в</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button4">
              <property name="text">
               <string>г</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button5">
              <property name="text">
               <string>д</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button6">
              <property name="text">
               <string>е</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button7">
              <property name="text">
               <string>ж</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button8">
              <property name="text">
               <string>з</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button9">
              <property name="text">
               <string>и</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button10">
              <property name="text">
               <string>к</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button11">
              <property name="text">
               <string>л</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button12">
              <property name="text">
               <string>м</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button13">
              <property name="text">
               <string>н</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button14">
              <property name="text">
               <string>о</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button15">
              <property name="text">
               <string>п</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetNoConstraint</enum>
            </property>
            <item>
             <widget class="QPushButton" name="button16">
              <property name="text">
               <string>р</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button17">
              <property name="text">
               <string>с</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button18">
              <property name="text">
               <string>т</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button19">
              <property name="text">
               <string>у</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button20">
              <property name="text">
               <string>ф</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button21">
              <property name="text">
               <string>х</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button22">
              <property name="text">
               <string>ц</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button23">
              <property name="text">
               <string>ч</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button24">
              <property name="text">
               <string>ш</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button25">
              <property name="text">
               <string>щ</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button26">
              <property name="text">
               <string>ы</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button27">
              <property name="text">
               <string>ь</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button28">
              <property name="text">
               <string>э</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button29">
              <property name="text">
               <string>ю</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button30">
              <property name="text">
               <string>я</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1632</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
